scripts/new_project.kts

```diff
@@ -37,14 +33,18 @@ object NewProject {
         Run kscript new_project.kts to create a new project with the following arguments:
             $KEY_PACKAGE_NAME=   New package name (i.e., com.example.package)
             $KEY_APP_NAME=       New app name (i.e., MyApp, "My App", "my-app")
-            $KEY_TEMPLATE=       Template (i.e., $TEMPLATE_XML, $TEMPLATE_COMPOSE)
+            $KEY_TEMPLATE=       Template (i.e. $TEMPLATE_COMPOSE)
```

Comment:
Since we only have one supported template now, we shouldn't need this argument anymore as the only possible value is `compose` 🤔

Reply:
@ryan-conway @thiennguyen0196 I think we can keep this argument as a common script to support new templates in the future. However, let's set its default value to `TEMPLATE_COMPOSE` and make it optional.